### PR TITLE
Specs customer settings

### PR DESCRIPTION
**Pull request by Saimis777**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: content/1.7/back-office/shop-parameters/customer-settings/view-groups.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+title: Groups
+weight: 1
+---
+
```
Comment (fabienvallon) [inline content/1.7/back-office/shop-parameters/customer-settings/view-groups.md:5]: Describe the page

File: content/1.7/back-office/shop-parameters/customer-settings/view-groups.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+### Call to action
```
Comment (fabienvallon) [inline content/1.7/back-office/shop-parameters/customer-settings/view-groups.md:6]: A new group button

File: content/1.7/back-office/shop-parameters/customer-settings/view-groups.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+Close: After clicking on it, the modal is closed
+Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+4. Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+# Groups 
+
+## Columns
```
Comment (fabienvallon) [inline content/1.7/back-office/shop-parameters/customer-settings/view-groups.md:44]: Add filters

File: content/1.7/back-office/shop-parameters/customer-settings/view-groups.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+Bulk actions can only be used on custom created groups, when they are created, only then bulk actions will appear.
+
+Clicking on the _Bulk actions_ button allows users to perform bulk actions for selected group (using the checkboxes). A user can:
+
+- Select all/Unselect all: this button will select/unselect all the checkboxes at the beginning of each row of the listing.
+- Delete selected: This button will delete the selected groups. 
+After clicking on "Delete selected", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" (See issue [#14462](https://github.com/PrestaShop/PrestaShop/issues/14462)). The action can be canceled by clicking on the cross or on the cancel button.
+After clicking on "Delete" button, the selected groups are deleted, a successful message is displayed "The selection has been successfully deleted.".
+
+# Default groups options
```
Comment (fabienvallon) [inline content/1.7/back-office/shop-parameters/customer-settings/view-groups.md:91]: Describe business rules (?)

**Review by fabienvallon — CHANGES_REQUESTED** [review]

File: content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+In this section the customer details which are already input will show, and all of them can be edited:
+
+1) Social title
+
+There are 2 options in this field, can select "Mr." or "Mrs.".
+
+2) First name
+
+This is a mandatory field, has a help text below: "Only letters and the dot (.) character, followed by a space, are allowed.". Trying to save with the field empty, will show a message by the field: "Please fill out this field". When trying to save with invalid characters, will show an error: "The "First name" field is invalid.".
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md:14]: All other characters are blocked, "-" is probably left for people who have 2 names as an example: Marie-henderson.

File: content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+3) Last name
+
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md:17]: All other characters are blocked, "-" is probably left for people who have 2 names as an example: Marie-henderson.

File: content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+This is a mandatory field, has a help text below: "Only letters and the dot (.) character, followed by a space, are allowed.". Trying to save with the field empty, will show a message by the field: "Please fill out this field". When trying to save with invalid characters, will show an error: "The "Last name" field is invalid.".
+
+4) Email
+
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md:23]: Yes, should contain a "@" and a dot for the extension.

File: content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+3 Drop-down fields are presented in this section for: Year, month (in numbers), date.
+
+7) Enabled
+
+This is a toggle switch, can be turned to "Yes" or "No". Has a help text below: "Enable or disable customer login.".
+
+8) Partner offers
+
+This is a toggle switch, can be turned to "Yes" or "No". Has a help text below: "This customer will receive your ads via email.".
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md:40]: @fabienvallon could you clarify? You mean the ads that are sent via email?

File: content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+ - Visitor
+ - Guest
+ - Customer
+ - And all custom ones
+
+10) Default customer group
+
+This is a drop-down list for the default customer group, the drop-down displays all the possible customer groups.
+
+Has a help text below: "This group will be the user's default group. Only the discount for the selected group will be applied to this customer."
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/edit-customer.md:61]: Could you clarify @fabienvallon vfat?

File: content/1.7/back-office/shop-parameters/customer-settings/view-groups.md (hunk 3 above)
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/view-groups.md:44]: Could you clarify what needs to be fixed here? I've described the filtering (search) and the sorting in the columns @fabienvallon

File: content/1.7/back-office/shop-parameters/customer-settings/view-customer-settings.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+This is an input field, that is measured by "minutes", by default the input is: 360. Has a help text below the field: "Minimum time required between two requests for a password reset.".
+Only numbers are allowed in this field, if any other characters are entered, when saving they are removed. 
+
+4) Enable B2B mode
+
+This is a mandatory toggle switch, by default it is set to: "No". Has a help text below: "Activate or deactivate B2B mode. When this option is enabled, B2B features will be made available.". 
+
+5) Ask for birth date 
+
+This is a mandatory toggle switch, by default it is set to: "No". Has a help text below: "Display or not the birth date field.".
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/view-customer-settings.md]: clarified now

File: content/1.7/back-office/shop-parameters/customer-settings/view-customer-settings.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+This is a mandatory toggle switch, by default it is set to: "Yes". Has a help text below: "Send an email with a summary of the account information after registration.". 
+
+3) Password reset delay
+
+This is an input field, that is measured by "minutes", by default the input is: 360. Has a help text below the field: "Minimum time required between two requests for a password reset.".
+Only numbers are allowed in this field, if any other characters are entered, when saving they are removed. 
+
+4) Enable B2B mode
+
+This is a mandatory toggle switch, by default it is set to: "No". Has a help text below: "Activate or deactivate B2B mode. When this option is enabled, B2B features will be made available.".
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/view-customer-settings.md:25]: can't find the B2B feature specs:/

File: content/1.7/back-office/shop-parameters/customer-settings/view-customer-settings.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+weight: 1
+---
+
+# General
+
+In this section 6 fields are presented:
+
+1) Re-display cart at login 
+
+This is a mandatory toggle switch, by default it is set to: "No". Has a help text below: "After a customer logs in, you can recall and display the content of his/her last shopping cart.".
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/view-customer-settings.md:12]: Could you clarify to where should I link this? The FO specs of the feature?

File: content/1.7/back-office/shop-parameters/customer-settings/view-customer-settings.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+# General
+
+In this section 6 fields are presented:
+
+1) Re-display cart at login 
+
+This is a mandatory toggle switch, by default it is set to: "No". Has a help text below: "After a customer logs in, you can recall and display the content of his/her last shopping cart.". 
+
+2) Send an email after registration
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/view-customer-settings.md:14]: The specs don't seem to exist unless they are not merged yet.

**Pull request merged**